web/app/components/doc/row.ts

```diff
@@ -0,0 +1,75 @@
+import Component from "@glimmer/component";
+
+interface DocRowComponentSignature {
+  Args: {
+    productArea: string;
+  };
+}
+
+export default class DocRowComponent extends Component<DocRowComponentSignature> {
+  get productAreaName() {
+    let productAreaName;
+
+    // TODO: Convert this to a switch statement
```

Comment:
I thought we had a helper for this logic somewhere 🤔

Reply:
Ah right, yeah we have one for the icon.

This one just needs to be cleaned up (and, separately, turned into its own helper).